src/components/textinput/_textinput.scss

```diff
@@ -70,4 +70,13 @@
 }
 textarea.cmp-adaptiveform-textinput__widget{
   height: 80px;
-}
\ No newline at end of file
+}
+.cmp-adaptiveform-textinput__charcounter {
+  margin-left: auto;
```

Comment:
use margin-inline-start as this will work in RTL as well.

Reply:
Hi @sakshi-arora1  fixed!